<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1193</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="wScatter" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>1151</width>
     <height>481</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>93</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOk">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>93</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>161</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>11</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>14</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="lblResponse">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Response:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbResponse"/>
        </item>
        <item>
         <widget class="QLabel" name="lblRate">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Learning Rate:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbRate">
          <item>
           <property name="text">
            <string>5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>0.1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>0.01</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>0.001</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblIter">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Max Iterations:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbIter">
          <item>
           <property name="text">
            <string>3000</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2500</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2000</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1500</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1000</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="wPredict" native="true">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>10</y>
     <width>271</width>
     <height>161</height>
    </rect>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>260</width>
      <height>141</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>6</number>
    </property>
    <widget class="QSplitter" name="splitter_4">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>10</y>
       <width>218</width>
       <height>51</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QLabel" name="lblPred">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Select predictor</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblPred_2">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Set value</string>
       </property>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QComboBox" name="cbPreds"/>
      <widget class="QDoubleSpinBox" name="dsbPreds">
       <property name="maximum">
        <double>10000.000000000000000</double>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QSplitter" name="splitter_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>188</width>
       <height>28</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QPushButton" name="btnSet">
      <property name="text">
       <string>Set</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPredict">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Predict</string>
      </property>
     </widget>
    </widget>
    <widget class="QCheckBox" name="cbClamp">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>151</width>
       <height>28</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Clamp predictions</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="wMssg" native="true">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>581</width>
     <height>151</height>
    </rect>
   </property>
   <widget class="QLabel" name="lblMssg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>571</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>Building a linear regression model...</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="wClamp" native="true">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>200</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lblClampMin">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Min:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblClampMax">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Max:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="cbMin"/>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="cbMax"/>
     </item>
    </layout>
   </widget>
  </widget>
  <zorder>wMssg</zorder>
  <zorder>wScatter</zorder>
  <zorder>btnCancel</zorder>
  <zorder>btnOk</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>wPredict</zorder>
  <zorder>wClamp</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
